packages/core/src/actions/smartAccount/buildUserOperationFromTxs.ts

```diff
@@ -20,10 +18,7 @@ export const buildUserOperationFromTxs: <
 >(
   client: Client<TTransport, TChain, TAccount>,
   args: SendTransactionsParameters<TAccount>
-) => Promise<BuildUserOperationFromTransactionsResult> = async (
```

Comment:
why not just update the type definition of `BuildUserOperationFromTransactionsResult`?

Reply:
wouldn't that be a breaking change?